### PR TITLE
#42 bound withs

### LINKED ISSUE #42: Bound-withs are bound to be an issue
In GitLab by @mxk128 on Sep 28, 2018, 14:42

How to we index/display Bound-with records. In Symphony these bound-with records don't contain itemids. Some catkeys examples:

```
1006921  
1021944
10018394
```

I will provide more examples.

### LINKED ISSUE #60: Setup testing suite

### PR DESCRIPTION
**Pull request by cdmo**
closes #42
closes #60

### CODE REVIEW

File: spec/lib/traject/title_spec.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                                                             { 't' => 'International organizations and competition law : diverging rationale? --' },
+                                                             { 't' => 'Market governance in China --' }] } }
+    end
+    let(:result) { @indexer.map_record(MARC::Record.new_from_hash('fields' => [fields], 'leader' => leader)) }
+
+    it 'returns with trailing -- chomped' do
+      expect(result[field]).not_to eq nil
+      expect(result[field]).to(all(not_include(' --')))
+    end
+  end
```
[the remark this answers is not in the document]
Comment (cdmo) [reply spec/lib/traject/title_spec.rb]: oh sure!

[the remark this answers is not in the document]
Comment (banukutlu) [reply spec/lib/traject/title_spec.rb]: per Charlie 
> title specs looks good

[the remark this answers is not in the document]
Comment (cdmo) [reply spec/lib/traject/title_spec.rb]: yes. I endorse this quote.

[the remark this answers is not in the document]
Comment (banukutlu) [reply spec/lib/traject/title_spec.rb]: good to hear @cdmo! LOL

**Pull request merged**